applications/sandbox/defaults/defaultpolicies.cpp

```diff
@@ -119,7 +119,7 @@ namespace legion::core
 
         auto scaleFactor = emitter.get_uniform<float>("scaleFactor");
 
-        if (!emitter.infinite)
+        if (!emitter.has_uniform<float>("minLifeTime") && !emitter.has_uniform<float>("maxLifeTime"))
```

Comment:
might wanna add a has_uniform that takes an id_type so you can precache the hash, and not have to hash it at run time